data/10/aria-marcello.mei

```diff
@@ -4,7 +4,7 @@
     <meiHead>
         <fileDesc>
             <titleStmt>
-                <title>Aria del Signore Marcello, Nobile Veneto.</title>
+                <title>Benedetto Marcello: Col pianto e coi (Aria)</title>
```

Comment:
As I pointed out in #6, the title in fileDesc should say about the file itself, and not just copy an original title from a source. However, this is not final, but a recommendation. We should come up with something that works equally across all files.